components/rhs_thread/rhs_thread.test.jsx

```diff
@@ -48,26 +48,32 @@ describe('components/RhsThread', () => {
 
     test('should match snapshot', () => {
         const wrapper = shallow(
-            <RhsThread {...baseProps}/>
+            <RhsThread {...baseProps}/>,
+            {disableLifecycleMethods: true},
```

Comment:
Disabling lifecycle method here because `componentDidUpdate` is calling the `ref` to `<Scrollbars/>`, throwing an `undefined` error.